projects/dsp-ui/src/lib/core/session.service.ts

```diff
@@ -0,0 +1,215 @@
+import { Inject, Injectable } from '@angular/core';
+import {
+    ApiResponseData,
+    ApiResponseError,
+    Constants,
+    CredentialsResponse,
+    KnoraApiConfig,
+    KnoraApiConnection,
+    UserResponse
+} from '@dasch-swiss/dsp-js';
+import { DspApiConfigToken, DspApiConnectionToken } from './core.module';
+
+/**
+ * Currently logged-in user information
+ */
+export interface CurrentUser {
```

Comment:
if `CurrentUser` is only used internally, there is no need to export it.

Reply:
Same as above. We need the information of the currently logged in user. It's important to know, if a user is system admin or not. And in which project a user is project admin.

Reply:
As the `CurrentUser` is part of `Session` we should export `Session` only.

Reply:
Resolved in d272070